homeassistant/components/light/deconz.py

```diff
@@ -12,6 +12,7 @@
     ATTR_TRANSITION, EFFECT_COLORLOOP, FLASH_LONG, FLASH_SHORT,
     SUPPORT_BRIGHTNESS, SUPPORT_COLOR, SUPPORT_COLOR_TEMP, SUPPORT_EFFECT,
     SUPPORT_FLASH, SUPPORT_TRANSITION, Light)
+from homeassistant.components.switch.deconz import SWITCH_TYPES
```

Comment:
A platform shouldn't depend on another platform. Better move this constant to the component package.